library/Fs.re

```diff
@@ -41,6 +41,27 @@ let exists = path => {
   };
 };
 
+// Credit: https://github.com/fastpack/fastpack/blob/9f6aa7d5b83ffef03e73a15679200576ff9dbcb7/FastpackUtil/FS.re#L94
+let rec rmdir = dir => {
+  let%lwt files = Lwt_stream.to_list(Lwt_unix.files_of_directory(dir));
```

Comment:
Consider using the pipe operator for readability (`Lwt_unix.files_of_directory(dir) |> Lwt_stream.to_list`)

Reply:
Addressed